activerecord/test/cases/fixtures_test.rb

```diff
@@ -73,8 +75,14 @@ def test_bulk_insert
       begin
         subscriber = InsertQuerySubscriber.new
         subscription = ActiveSupport::Notifications.subscribe("sql.active_record", subscriber)
-        create_fixtures("bulbs")
+        create_fixtures("bulbs", "authors", "computers")
+
         assert_equal 1, subscriber.events.size, "It takes one INSERT query to insert two fixtures"
+        if current_adapter?(:Mysql2Adapter)
+          assert_match(/INSERT INTO `bulbs`.*INSERT INTO `authors`.*INSERT INTO `computers`/m, subscriber.events.first)
```

Comment:
This loses [the assertion of] the previous bulk behaviour: that it all happens in a single INSERT per table, and not one per row -- this would pass even if we were sending a huge stream of single-row INSERTs.

Reply:
👍 , I decided to write a separate test instead